backend/capellacollab/sessions/operators/k8s.py

```diff
@@ -132,10 +159,50 @@ def start_persistent_session(
             tool_name=tool_name,
             version_name=version_name,
             environment=environment,
+            ports={"rdp": 3389, "metrics": 9118},
             persistent_workspace_claim_name=self._get_claim_name(username),
             pure_variants_secret_name=pure_variants_secret_name,
         )
 
+    def start_persistent_jupyter_session(
+        self,
+        username: str,
+        tool_name: str,
+        version_name: str,
+        token: str,
+        docker_image: str,
+    ) -> dict[str, t.Any]:
+        self._create_persistent_volume_claim(username)
+
+        path = f"/jupyter/{username}"
+
+        environment = {
+            "JUPYTER_BASE_URL": path,
+            "JUPYTER_TOKEN": token,
+            "JUPYTER_PORT": "8888",
+        }
+
+        session_parameters = self._start_session(
+            image=docker_image,
+            username=username,
+            session_type="persistent",
+            tool_name=tool_name,
+            version_name=version_name,
+            environment=environment,
+            ports={"http": 8888},
```

Comment:
You use the port `8888` at five different places here. So I think we should add a `JUPYTER_PORT` at the top of the k8s operator and then simply use this here. This makes it a lot easier to make changes to it later (e.g., if we want to make it configurable or if we want to change the port for some reason)

Reply:
Discussed with @dominik003. Let's leave it for now. We need more cleanup in the k8s operator anyway.